### PR TITLE
[Ot171-39] Testimonials Slider

### PR DESCRIPTION
**Pull request by frannasich**
### Slider Testimonios changes description:
-  ScrollView added and slider Testimonios added
-  Added cases of slides
-  Added TestimoniosCollectionViewCell
-  Some images added
-  Testimonios max 4 pages added
-  Added SeeMoreCollectionViewCell

https://user-images.githubusercontent.com/80861248/161852712-b4bd7fbc-a22c-452b-9d4b-0a5a99b585c9.mov

**Jira Ticket**
[OT171-39](https://alkemy-labs.atlassian.net/jira/software/c/projects/OT171/boards/247?modal=detail&selectedIssue=OT171-39)

### CODE REVIEW

File: SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift [hunk 1]
```diff
@@ -15,15 +15,25 @@ class HomeViewController: UIViewController {
         let image: UIImage?
     }
     
+    struct TestimonialsData {
```
Comment (gapoj) [inline SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift:18]: this should be in a separate file

File: SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/TestimonialsCollectionViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import UIKit
+
+class TestimonialsCollectionViewCell: UICollectionViewCell {
+
+    
+    @IBOutlet weak var testimonialImage: UIImageView!
+    @IBOutlet weak var testimonialEpigraph: UILabel!
+    
+    override func awakeFromNib() {
```
Comment (gapoj) [inline SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/TestimonialsCollectionViewCell.swift:16]: you don´t need to override a function if you do nothing different that in the parent views

File: SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift [hunk 3]
```diff
@@ -32,37 +42,86 @@ class HomeViewController: UIViewController {
         collectionView.delegate = self
         collectionView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
         collectionView.register(UINib(nibName: "HomeCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "mycell")
-        
+        testimonialsCollectionView.isPagingEnabled = true
+        testimonialsCollectionView.dataSource = self
+        testimonialsCollectionView.delegate = self
+        testimonialsCollectionView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
```
Comment (gapoj) [inline SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift:48]: you should not add code that do nothing

**Review by gapoj — APPROVED** [review]
I will approve the PR but pay attention to the details and the comments I put on it

File: SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/seeMoreCollectionViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,16 @@
+//
+//  SeeMoreCollectionViewCell.swift
+//  SomosMasApp
+//
+//  Created by Fran Nasich on 05/04/2022.
+//
+
+import UIKit
+
+class SeeMoreCollectionViewCell: UICollectionViewCell {
```
[the remark this answers is not in the document]
Comment (frannasich) [reply SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/seeMoreCollectionViewCell.swift:10]: I changed it, but in Github nothing is changed about the name of the file... but the name of the class if I change well

**Comment by gapoj** [comment]
PR WITH CONFLICTS, is not mergeable, Is not acceptable

**Review by gapoj — CHANGES_REQUESTED** [review]
PR STILL HAVE CONFLICTS

**Review by gapoj — APPROVED** [review]
I will merge it but pay attention to the reviews

**Pull request merged**